CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required (VERSION 3.17)
+cmake_policy (SET CMP0053 NEW)
+cmake_policy (SET CMP0054 NEW)
+
+project (
+        GOCART
+        VERSION 10.12.5
+        LANGUAGES Fortran CXX C)  # Note - CXX is required for ESMF
+
+if ("${PROJECT_SOURCE_DIR}" STREQUAL "${PROJECT_BINARY_DIR}")
+    message(SEND_ERROR "In-source builds are disabled. Please
+           issue cmake command in separate build directory.")
+endif ("${PROJECT_SOURCE_DIR}" STREQUAL "${PROJECT_BINARY_DIR}")
+
+# Set the default build type to release
+if (NOT CMAKE_BUILD_TYPE)
+    message (STATUS "Setting build type to 'Release' as none was specified.")
+    set (CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
+    # Set the possible values of build type for cmake-gui
+    set_property (CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS
+            "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
+endif ()
+
+set (DOING_GEOS5 YES)
```

Comment:
My guess is this might be moot. I think it's only used in @GEOSchem_GridComp/Shared/Chem_Shared:
```cmake
if (DOING_GEOS5)
  list (APPEND srcs mod_diag.F90 diaglist.F90)
  list (APPEND srcs Chem_UtilMod.F90 Chem_StateMod.F90)
else ()
  list (APPEND srcs Chem_UtilMod.F90)
  list (APPEND srcs Chem_UtilMod_geos4.F90)
endif ()
```

It's there because it was there in GNU make/CVS land. If @mmanyin or @adarmenov say it's no longer needed, we can nuke that code all together.

Reply:
I have scanned the GCM and CTM code (*F90) for Chem_UtilMPread; it is not used, so the geos4 code appears superfluous.  If @adarmenov agrees, we could just change the section above to
```
  list (APPEND srcs mod_diag.F90 diaglist.F90)
  list (APPEND srcs Chem_UtilMod.F90 Chem_StateMod.F90)
```
without checking  DOING_GEOS5.

Reply:
Agreed.

Reply:
Note - that the main copy in GEOSchem_GridComp should get this fix.